avm/res/insights/scheduled-query-rule/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.29.47.4906",
-      "templateHash": "12034267134047308763"
+      "version": "0.26.170.59819",
```

Comment:
Hey @emichellecarter, beware the downgrade of the Bicep version. As there have been quite some changes to the formatting since `0.28`, I'd recommend to upgrade your version and re-run the build again.

The important one in the Bicep CLI you can find via `bicep --version` which needs upgrading.

After you upgraded you can either run the `Set-AVMModule` script again, or simply run `bicep build <path>` on the module's `main.bicep` file

Reply:
Updated.  Thank you for catching this and the guidance for the fix.